dressup/src/components/Buttons/PrimaryNavbutton.js

```diff
@@ -0,0 +1,13 @@
+import "./PrimaryNavbutton.css";
+
+function PrimaryNavbutton(props) {
+  return (
```

Comment:
suggestion: instead of props, destructure the props and use it like " const PrimaryNavButtom = ({ text }) => { ... } "